docs/upgrade.html

```diff
@@ -39,22 +39,29 @@
+    <li> Once the entire cluster is upgraded, bump the protocol version by editing inter.broker.protocol.version and setting it to 0.11.0. </li>
+    <li> If your previous message format is 0.10.0, change log.message.format.version to 0.11.0 (this is a no-op as the message format is the same for 0.10.0, 0.10.1, 0.10.2 and 0.11.0).
         If your previous message format version is lower than 0.10.0, do not change log.message.format.version yet - this parameter should only change once all consumers have been upgraded to 0.10.0.0 or later.</li>
     <li> Restart the brokers one by one for the new protocol version to take effect. </li>
     <li> If log.message.format.version is still lower than 0.10.0 at this point, wait until all consumers have been upgraded to 0.10.0 or later,
-        then change log.message.format.version to 0.10.3 on each broker and restart them one by one. </li>
+        then change log.message.format.version to 0.11.0 on each broker and restart them one by one. </li>
 </ol>
 
 <p><b>Note:</b> If you are willing to accept downtime, you can simply take all the brokers down, update the code and start all of them. They will start with the new protocol by default.
 
 <p><b>Note:</b> Bumping the protocol version and restarting can be done any time after the brokers were upgraded. It does not have to be immediately after.
 
-<h5><a id="upgrade_1030_notable" href="#upgrade_1030_notable">Notable changes in 0.10.3.0</a></h5>
+<h5><a id="upgrade_1100_notable" href="#upgrade_1100_notable">Notable changes in 0.11.0.0</a></h5>
 <ul>
-    <li>The <code>offsets.topic.replication.factor</code> broker config is now enforced upon auto topic creation. Internal auto topic creation will fail with a GROUP_COORDINATOR_NOT_AVAILABLE error until the cluster size meets this replication factor requirement.</li>
-    <li>By default <code>message.timestamp.difference.max.ms</code> is the same as <code>retention.ms</code> instead of <code>Long.MAX_VALUE</code>.</li>
+    <li>The <code>offsets.topic.replication.factor</code> broker config is now enforced upon auto topic creation. Internal
+        auto topic creation will fail with a GROUP_COORDINATOR_NOT_AVAILABLE error until the cluster size meets this
+        replication factor requirement.</li>
+    <li>By default <code>message.timestamp.difference.max.ms</code> is the same as <code>retention.ms</code> instead of
+        <code>Long.MAX_VALUE</code>.</li>
+    <li>Beginning in 0.11.0, the configuration <code>max.message.bytes</code> applies to the total size of a batch of messages.
+        Previously the setting applied to batches of compressed messages, or to non-compressed messages individually. In practice,
+        the change is minor since a message batch may consist of only a single message, so the limitation on the size of
+        individual messages is only reduced by the overhead of the batch format.</li>
```

Comment:
Is it worth saying something about `batch.size`? It seems that is what ensures that the batch will be of an appropriate size.